### PR TITLE
Added max block size and max single put size

### PR DESCRIPTION
**Pull request by mganesh1308**

### CODE REVIEW

**Comment by crepererum** [comment]
Depends on mbr/simplekv#97

**Comment by DD5HT** [comment]
Do you have a benchmark, to show the improvement? Also please squash your commits. Otherwise it looks good.

File: storefact/__init__.py [hunk 1]
```diff
@@ -54,7 +56,9 @@ def get_store(type, create_if_missing=True, **params):
       ``"create_if_missing"`` has to be ``False`` if ``"use_sas"`` is set. When ``"use_sas"`` is set,
       ``"account_key"`` is interpreted as Shared Access Signature (SAS) token.FIRE
       ``"max_connections"``: Maximum number of network connections used by one store (default: ``2``).
-      ``"socket_timeout"``: maximum timeout value in seconds (socket_timeout: ``100``).
+      ``"socket_timeout"``: maximum timeout value in seconds (socket_timeout: ``200``).
+      ``"max_single_put_size"``: max_single_put_size is the largest size upload supported in a single put call.
+      ``"max_block_size"``: maximum block size is maximum size of the blocks(maximum size is >= 100MB)
```
Comment (DD5HT) [inline storefact/__init__.py]: do you mean `<= 100MB`?

**Comment by crepererum** [comment]
> Do you have a benchmark, to show the improvement? Also please squash your commits. Otherwise it looks good.

Introducing the config option per se does not change anything, but it is required to actually enable users to run some proper benchmarks and use a better option than the default, depending on the concrete use case.

**Review by DD5HT — APPROVED** [review]

**Pull request merged**